EnglishFilter/sections/antiadblock.txt

```diff
@@ -1708,7 +1710,8 @@ claimtrx.club#%#//scriptlet("set-constant", "show_ads", "true")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56627
 leakite.com#%#//scriptlet("abort-on-property-write", "ai_check_block")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56588
-key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115
+deportealdia.live,key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

Comment:
Remove `deportealdia.live,` and return the original task `! https://github.com/AdguardTeam/AdguardFilters/issues/56588`.